products/opensuse/main.pm

```diff
@@ -370,6 +370,31 @@ else {
             loadtest "network/wireguard";
             return 1;
         }
+        if (get_var('OVS')) {
+            set_var('INSTALLONLY', 1);
+            if (check_var('HOSTNAME', 'server')) {
+                barrier_create('ipsec_done',          2);
+                barrier_create('traffic_check_done',  2);
+                barrier_create('password_ok',         2);
+                barrier_create('ipsec1_done',         2);
+                barrier_create('traffic_check_done1', 2);
+                barrier_create('ipsec2_done',         2);
+                barrier_create('traffic_check_done2', 2);
+                barrier_create('cert_done',           2);
+                barrier_create('host2_cert_ready',    2);
+                barrier_create('cacert_done',         2);
+            }
+            loadtest 'installation/bootloader_start';
```

Comment:
Isn't here missing `boot_hdd_image;`, same as in `sle/main.pm` and VR ?

Reply:
boot_hdd_image is already running in the condition `elsif (get_var("BOOT_HDD_IMAGE")` which includes the schedule for ovs-client and ovs-server. It just seems that the main.pm for sle and opensuse have some deviations in the structure ^^